src/Files.Backend/Services/IPreviewPopupService.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Files.Backend.Services
+{
+	public interface IPreviewPopupService : INotifyPropertyChanged
+	{
+		/// <summary>
+		/// Opens preview popup
+		/// </summary>
+		/// <param name="path"></param>
+		/// <param name="switchPreview"></param>
+		/// <returns></returns>
+		Task OpenPreviewPopup(string path, bool switchPreview = false);
```

Comment:
Shouldn't SwitchPreview be a separate API?

Reply:
The only difference (at least with QuickLook) is the message so it can help reduce duplicate code. If you think it's worth it, I can split them up.

Reply:
It looks like SeerPro works differently so it'll make sense to split it.

Reply:
Yes I think they should be split. It is confusing that the popup doesn't actually open when switchPreview is true.

Reply:
Done